audit_engine/scoring/scoring_engine.py

```
@@ -0,0 +1,46 @@
+Minimal scoring engine used by AuditEngine._apply_scoring.
+
+Provides a calculate_score(Finding) API with robust severity mapping.
+"""
+
+from __future__ import annotations
+
+from typing import Any
+
+try:
+    from audit_engine.core.schemas import SeverityLevel  # type: ignore
+except Exception:  # pragma: no cover
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)